.rubocop_todo.yml

```diff
@@ -437,12 +412,6 @@ Style/OpMethod:
 Style/ParallelAssignment:
   Enabled: false
 
-# Offense count: 37
-# Cop supports --auto-correct.
-# Configuration parameters: AllowSafeAssignment.
```

Comment:
This one is made specifically so that we don't make stupidly long conditions, and extract these to methods that are more readable. Why is that a bad thing? We're a few steps from fixing it completely.